buildSrc/buildUtils.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * This file contains some utilities used from various build scripts in this directory.
+ */
+import fs from "fs-extra"
+import path from "path"
+
+/** global used by the measure() function to mark the start of measurement **/
+var measureStartTime
+
+/**
+ * Returns tutanota app version (as in package.json).
+ * @returns {Promise<*>}
+ */
+export async function getTutanotaAppVersion() {
+	const {version} = JSON.parse(await fs.readFile("package.json", "utf8"))
```

Comment:
I think this will break if it gets called from below the root directory, like in `test/`?

Reply:
In some other places too

Reply:
As far as I understand it "package.json" will be resolved in relation to the CWD. Since we (as far as I know) start our scripts from the project's root directory, it should be fine. Feel free to suggest a safer solution.

Reply:
i often run `test.js` directly from within `test/`. It doesn't cause any issues at the moment but there is a way to load files relative to the script rather than cwd. I will make the change now just because it's easy and futureproof